types/config.d.ts

```diff
@@ -64,6 +64,8 @@ declare global {
          * Ignore means a failure is expected and no action should be taken.
          */
         loadFailureMode?: 'fatal'|'warn'|'ignore';
+        /** Whether the pass should skip the page load. If false, gathering begins as soon as navigation completes. */
```

Comment:
```suggestion
        /** Whether the pass should skip waiting for the page to load. If true, gathering begins as soon as navigation completes. */
```
Is it `s/false/true`? If it is supposed to be false, maybe reword for clarity?